quint/src/types/printing.ts

```diff
@@ -28,6 +29,13 @@ export function constraintToString(c: Constraint): string {
  * @returns a string with the formatted type scheme
  */
 export function typeSchemeToString(t: TypeScheme): string {
+  const [vars, type] = canonicalTypeScheme(t)
+
+  const varsString = vars.length > 0 ? `∀ ${vars.join(', ')} . ` : ''
```

Comment:
Also here, I would propose to use a more conventional syntax like `[x, y, z]. type`